universal-application-tool-0.0.1/conf/messages.en

```diff
@@ -0,0 +1,3 @@
+apply=Apply
+programs=Programs
+nextBlock=Save and continue
```

Comment:
Question (blocking): how might we organize these files so they don't become a mess? Is it possible to put comments in these files? Could we add sections for different parts of the UI e.g. a COMPONENTS section with the different button texts, a PAGES section with sub-sections for different pages?

Reply:
Great point! I added comments to `messages.en` - this is the default and the one we should keep the most organized. Thoughts?

Reply:
Looks great!